proto/v1/v1/database_service.proto

```diff
@@ -784,7 +784,8 @@ message ColumnMetadata {
 
   // The default value of column.
   bool default_null = 4;
-  string default_string = 5;
+  string default_string = 5 [deprecated = true];
```

Comment:
I think we can make breaking change for the API change. Anyway, let's move forward for now.